File: bronze/1546.py
n = int(input())
a = list(map(int, input().split(" ")))

max = 0
sum = 0
for i in range(0, n):
    if(a[i] > max): max = a[i]

for i in range(0, n):
    sum += a[i] / max * 100

print(sum / n)

File: bronze/1546.c
#include <stdio.h>

int main() {
    int a = 0, n = 0;
    scanf("%d", &n);

    double sum = 0;
    for(int i = 0 ; i<n; i++){
        scanf("%d", &a);
        sum += a;
    }

    printf("%.8lf", sum / n);
}